### ISSUE TITLE
Speedhack audio

### ISSUE THREAD

**Issue by ItzGalaxyGaming** [absoIute/Mega-Hack-Pro-Future#45]
Yo, I wanna know if it is possible to put an option that the "Speedhack audio" doesnt gives that Nightcore/Daycore effect.

**Comment by Brittank88** [comment]
Currently there's no option for this (as far as I'm aware) - this is actually somewhat of an interesting problem to solve.

**Comment by Brittank88** [comment]
> since it'd only work between 0.5x & 2x speed

If you implemented that workaround is this still true?

**Comment by absoIute** [comment]
Yeah, by default the calculation needed would just be 1/speed since pitch args are logarithmic (0.5 = -1 octave, 1 = default, 2 = +1 octave) but that's feels weird so i just use a linear octave scale

**Comment by Brittank88** [comment]
> but that's feels weird

What feels weird about it? If it maps out the correct behaviour isn't that what it should be?

I'm just hoping to understand this better.

**Comment by absoIute** [comment]
Basically since it's limited to one octave you get a minimum of 0.5 and a maximum of 2.0, going any lower/higher won't make any changes

If I can get it to pitch shift without affecting quality as much I'll most likely switch it to its standard scale though
